src/main/java/org/jabref/gui/groups/GroupDialogViewModel.java

```diff
@@ -310,10 +332,15 @@ public AbstractGroup resultConverter(ButtonType button) {
         try {
             String groupName = nameProperty.getValue().trim();
             if (typeExplicitProperty.getValue()) {
-                resultingGroup = new ExplicitGroup(
+                ExplicitGroup tempResultingGroup = new ExplicitGroup(
```

Comment:
Why a temp group?

You can directly add to a group?

Please change back and just add to `resultingGroup`

Reply:
`add` is only on `EntriesChanger`, which `ExplicitGroup` implements but `AbstractGroup` does not. I used the `temp` rather than casting for adding on line 341, but can cast if you prefer.